### DISCUSSION TITLE
How to deploy on canary enabled stage?

### DISCUSSION THREAD

**Discussion by shubsaini09** [serverless/serverless#11482] [Q&A]
While trying to deploy the stack, I am getting below error

```
An error occurred: ApiGatewayDeployment16669 - Resource handler returned message: "Invalid deployment content specified.Non canary deployment not supported on stage v1 because Canary is enabled. Provide CanarySettings in deploymentInput. (Service: ApiGateway, Status Code: 400, Request ID: a6e49347-2b63-a203-59e239c3)" (RequestToken: 8368f899-86fb-532d8492, HandlerErrorCode: InvalidRequest).
```
If i delete my canary stage and deploy then it works but this does not seem to me an appropriate option. How can i resolve this using serverless?

**Comment by hanhdt** [comment]
I have the same issue. Does anyone have suggestions for it?

**Comment by ewaty** [comment]
This problem goes away when you switch to the old console. Unfortunately they are phasing it out by Oct 2023, so one can only hope that they'll address it somehow :(
